recipes/xkcd-cli/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "xkcd-cli" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xkcd-cli-{{ version }}.tar.gz
```

Comment:
Do we want to inline `name` everywhere or use the Jinja variable?

No strong feelings either way. Just wanted to ask given we seem to be doing both